cmd/zapi/cmd/cli.go

```diff
@@ -8,8 +8,9 @@ import (
 	"os"
 	"syscall"
 
+	"github.com/brimsec/zq/api"
 	"github.com/brimsec/zq/cli"
-	"github.com/brimsec/zq/zqd/api"
+	"github.com/brimsec/zq/cmd/zapi/connection"
```

Comment:
I think we should keep all this stuff in the api package, but if we do move it, I think it ought to move to github.com/brimsec/zq/api/client.

Reply:
I would like to move it out of `api`, to remove as many dependencies of that package as we can (for example, the resty client). I'm fine with moving it to `api/client`.